Dockerfile

```
@@ -1,5 +1,5 @@
 ## Global Args #################################################################
-ARG BASE_UBI_IMAGE_TAG=9.3-1610
+ARG BASE_UBI_IMAGE_TAG=9.4-1123.1719560047
```

Can we just set this to `9.4`? 

```suggestion
ARG BASE_UBI_IMAGE_TAG=9.4
```

---

done